### ISSUE TITLE
web page layout totally broken

### ISSUE THREAD

**Issue by gunchleoc** [widelands/widelands_website#3]
The web page layout is totally broken and has been so for a long time. The actual content does not begin at the left but only after the last entry in the top menu. See attached screenshot. Similar in Firefox.

**Comment by gunchleoc** [comment]
(by sigra)

**Comment by gunchleoc** [comment]
(by sirver)
It looks good in webkit (safari) and firefox (3) on the mac. Can you provide details to where the error lies?

**Comment by gunchleoc** [comment]
(by sirver)
I fixed some (imho uncritical) bugs reported by validation. There are now only 4 bugs left concerning the search line which are definitively  uncritical and degrade well. Is the layout now less broken?

**Comment by gunchleoc** [comment]
(by sigra)
wl.widelands.org looks the same. Byakushin reported a similar problem when increasing font size.

**Comment by gunchleoc** [comment]
(by kataja)

**Comment by gunchleoc** [comment]
(by kataja)

**Comment by gunchleoc** [comment]
(by kataja)
Included two more screenshots. The first issue with the top menu not reacting nicely to lack of absolute horizontal space is visible both on my old FF and intrepid's newer FF. On the other hand, the page becomes unusable from a font size increase only in the older FF (or this size font/window - this is basically my laptop configuration, and the newer is in a desktop set), as seen in the latter screenshot. (In the newer one it just looked more like the first shot.)

wl.widelands.org behaves the same, I just noticed when I tried to read the forums and had to increase the font on my small screen.
